frontend/src/features/videos/VideoList.tsx

```diff
@@ -1,46 +1,35 @@
 import React from 'react';
 
-import { Typography, makeStyles, Grid } from '@material-ui/core';
-import type {
-  PaginatedVideoSerializerWithCriteriaList,
-  VideoSerializerWithCriteria,
-} from 'src/services/openapi';
+import { Typography, Box } from '@material-ui/core';
 import VideoCard from '../videos/VideoCard';
 import { CompareNowAction, AddToRateLaterList } from 'src/utils/action';
 import { useLoginState } from 'src/hooks';
+import { ActionList, VideoObject } from 'src/utils/types';
 
-const useStyles = makeStyles(() => ({
-  card: {
-    alignItems: 'center',
-  },
-}));
+interface Props {
+  videos: VideoObject[];
+  settings?: ActionList;
+}
 
-function VideoList({
-  videos,
-}: {
-  videos: PaginatedVideoSerializerWithCriteriaList;
-}) {
-  const classes = useStyles();
+function VideoList({ videos, settings = [] }: Props) {
   const { isLoggedIn } = useLoginState();
 
   const actions = isLoggedIn ? [CompareNowAction, AddToRateLaterList] : [];
 
   return (
-    <div>
-      {videos.results?.length ? (
-        videos.results.map((video: VideoSerializerWithCriteria) => (
-          <Grid container className={classes.card} key={video.video_id}>
-            <Grid item xs={12}>
-              <VideoCard video={video} actions={actions} />
-            </Grid>
-          </Grid>
+    <>
+      {videos.length ? (
+        videos.map((video: VideoObject) => (
+          <Box key={video.video_id} margin={1}>
```

Comment:
You have reduced the margin between the video cards on the recommendation page and I think it's going the opposite way as what we should do. I remember Laure reporting that it lacks a bit of space. I appreciate the extra padding though.